cloudify_cli/tests/commands/test_teardown.py

```diff
@@ -22,6 +24,7 @@ def test_teardown_has_existing_deployments_ignore_deployments(self, mock_teardow
         )
         self.invoke('cfy use 10.0.0.1 -m admin -p admin')
         env.profile = env.get_profile_context(suppress_error=True)
+        sys.modules['cloudify_cli.commands.teardown'].profile = env.profile
```

Comment:
Wouldn't this line effect other tests? perhaps the with patch(...) pattern should be used here?